x/authenticator/ante/ante.go

```diff
@@ -101,11 +101,10 @@ func (ad AuthenticatorDecorator) AnteHandle(
 		}
 	}()
 
-	// This is a transient context stored globally throughout the execution of the tx
-	// Any changes will to authenticator storage will be written to the store at the end of the tx
-	cacheCtx := ad.authenticatorKeeper.TransientStore.ResetTransientContext(ctx)
+	cacheCtx, writeCache := ctx.CacheContext()
+
 	// Ensure that no usedAuthenticators are stored in the transient store
```

Comment:
The comment still mention transient store

Reply:
The comment was removed.